target/linux/ath79/dts/qca9558_belkin_f9j1108-v2.dts

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	model = "Belkin AC1750 DB Wi-Fi (F9J1108v2)";
```

Comment:
I would invert the naming here, so the primary name matches the compatible:
Belkin F9J1108 v2 (AC1750 DB Wi-Fi)